### PR TITLE
DNS setup guidance

### PR DESCRIPTION
**Pull request by ian-shim**
Adding guidance to set up DNS address as operator socket and disable auto-IP-discovery

### CODE REVIEW

File: docs/eigenda/operator-guides/operator-faq.md [hunk 1]
```diff
@@ -61,12 +61,13 @@ out](./overview#eigenda-churn-approver) by another operator or you have been
...
 
-If you are running on k8s or have a load balancer in front of your EigenDA node
+If you have a static IP address or DNS address set up to receive the traffic 
+(i.e. running on k8s or have a load balancer in front of your EigenDA node)
 and you don't want EigenDA to automatically update IP which is sent to EigenDA
-while registering, then follow the steps to make sure correct IP is registered
+while registering, then follow the steps to make sure correct IP is registered:
 
-* Update the [NODE_HOSTNAME](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L63) to the public IP where you will want to recieve traffic.
+* Update the [NODE_HOSTNAME](https://github.com/Layr-Labs/eigenda-operator-setup/blob/v0.6.0/holesky/.env.example#L71) to the public IP where you will want to recieve traffic.
```
Comment (ian-shim) [inline docs/eigenda/operator-guides/operator-faq.md:71]: This needs to be updated when `.env.example` is added in `mainnet` directory

Comment (wesfloyd) [reply docs/eigenda/operator-guides/operator-faq.md:71]: @shrimalmadhur do you prefer to propose a PR for the mainnet .env.example file? Or I can propose a PR for it if you like

**Review by wesfloyd — APPROVED** [review]
lgtm

File: docs/eigenda/operator-guides/operator-faq.md (hunk 1 above)
Comment (eigensid) [reply docs/eigenda/operator-guides/operator-faq.md:71]: @wesfloyd Pls cut a PR

**Pull request merged**